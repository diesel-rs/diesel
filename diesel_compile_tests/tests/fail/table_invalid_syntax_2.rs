#[macro_use]
extern crate diesel;

table! {
     some wrong syntax
     //~^ ERROR: expected one of `!` or `::`, found `wrong`
}

fn main() {}
