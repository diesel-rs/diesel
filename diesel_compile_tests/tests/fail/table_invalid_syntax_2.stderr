error: expected one of `!` or `::`, found `wrong`
 --> tests/fail/table_invalid_syntax_2.rs:5:11
  |
LL |      some wrong syntax
  |           ^^^^^ expected one of `!` or `::`
